### ISSUE TITLE
Keyboard shortcuts still work when in tablet mode

### ISSUE THREAD

**Issue by KNejad** [ragtag/spin#4]
When I am in tablet mode no keys work (as expected) but if I try a keyboard shortcut it still works. I like this because it means I can easily switch between tablet and laptop mode, but it also means that I might accidentally pause my music when in tablet mode.

**Comment by ragtag** [comment]
Yes, the keyboard still works in tablet mode. It's not actually spin.py that is controlling the keyboard. The keyboard gets disabled when you flip the screen around physically, and works out of the box. 

What spin.py does when it switches to tablet mode, is turn on the motion sensor, so that when you rotate the screen physically, the image is rotated too (and the pen and touch sensors), and it disables the trackpad.

This actually gave me an idea. I've tried to get the machine to automatically switch to tablet mode when you flip the screen around, but not found a good solution for this. But since the keyboard works, maybe there is some way I can detect if the keyboard has been disabled, and then enter tablet mode. Hmm...need to do some research.
